api_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/bot-api/telegram"
-	"github.com/m0sth8/httpmock"
+	"github.com/jarcoal/httpmock"
```

Comment:
I didn't find additional commits in `m0sth8/httpmock` fork so I decided to switch it to upstream because they have a very useful feature `httpmock.GetTotalCallCount`